packages/@react-aria/textfield/src/useTextField.ts

```diff
@@ -79,12 +79,7 @@ export interface AriaTextFieldOptions<T extends TextFieldIntrinsicElements> exte
    * For example, [`type`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#attr-type).
    * @default 'input'
    */
-  inputElementType?: T,
-  /**
-   * A nonstandard attribute used by iOS Safari that controls how textual form elements should be automatically capitalized.
-   * See [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#autocapitalize).
-   */
-  autoCapitalize?: 'off' | 'none' | 'on' | 'sentences' | 'words' | 'characters'
```

Comment:
Indeed, as mentioned in the comment, the property should have been added to `packages/@react-types/shared/src/dom.d.ts`.
https://github.com/adobe/react-spectrum/blob/ee403ec3ba5dba7c66903e3454b8f510c628a8f9/packages/%40react-types/shared/src/dom.d.ts#L133-L134.